examples/src/main/AndroidManifest.xml

```diff
@@ -15,9 +15,9 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
 
-        <!-- Uncomment this line of code if you want to use MAU based billing
+        <!-- Uncomment this line of code if you want to use MAU based billing-->
         <meta-data android:name="com.mapbox.ManageSkuToken" android:value="false" />
```

Comment:
This is still an open question for Andrew Chen. If we allow no SKU, then you basically don't need to enable it. Until you enable the MAU flag you are using no SKU by default. Does that makes sense

Reply:
I believe we should move this under `libnavigation-core` `AndroidManifest` now as it's uncertain if we want to allow no SKU, but we know for sure that MAU-based billing is the default. What do you think?

cc @andrewychen

Reply:
Since, until now it was unclear if we are allowing no SKU and if MAU based billing will be default, I chose to add this tag to `examples/AndroidManifest.xml`. We have confirmed that MAU will be default and client will be allowed to switch to no sku. I am going to move this tag to `libnavigation-core/AndroidManifest.xml`